dependency-manager-tests/carthage/CTProject.xcodeproj/project.pbxproj

```diff
@@ -14,8 +14,6 @@
 		61C36425243752A600C4D4E6 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 61C36423243752A600C4D4E6 /* LaunchScreen.storyboard */; };
 		61C36430243752A600C4D4E6 /* CTProjectTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 61C3642F243752A600C4D4E6 /* CTProjectTests.swift */; };
 		61C3643B243752A600C4D4E6 /* CTProjectUITests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 61C3643A243752A600C4D4E6 /* CTProjectUITests.swift */; };
-		9E9D5E8825F90FC6002F12A0 /* Kronos.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = 9E9D5E8525F90FC6002F12A0 /* Kronos.xcframework */; };
```

Comment:
this is actually a breaking change for our carthage users.
ideally this would require a major version update.
we should at least communicate this in release notes.

Reply:
Indeed 👍, Carthage users will have to remove `Kronos.xcframework` from their list of linked libraries, otherwise it will fail on first clean build. Although, I don't think it's a breaking change requiring major update (well, it's Carthage limitation), but it's definitely something we need to mention under "Migration Steps (Carthage)" in Release Notes for `1.9.0` 👌.

Reply:
Added `needs-migration-steps-explained` label.